funnel/templates/proposal.html.jinja2

```diff
@@ -23,17 +23,19 @@
   <h1 class="mui--text-display1"><a href="{{ project.url_for() }}"><strong>{{ project.title }}</a></strong></h1>
   <p class="mui--text-headline">{{ project.tagline }}</p>
   <p>
-  <a class="mui-btn mui-btn--primary mui--d-inlineblock" href="{{ project.url_for() }}#tickets" data-action="Tickets(header-proposalpage)">
-    {%- if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
-      {% if project.boxoffice_data.buy_btn_text -%}
-        {{ project.boxoffice_data.buy_btn_text }}
-      {% else %}
-        {% trans %}Tickets{% endtrans %}
-      {% endif %}
-    {%- else -%}
-      {% trans %}Attend{% endtrans %}
-    {%- endif %}
-  </a>
+  {% if project.features.has_tickets() or project.features.has_rsvp() %}
```

Comment:
Make this a new feature named `has_tickets_or_rsvp`. It can contain:

```python
return obj.features.has_tickets() or obj.features.has_rsvp()
```

Reply:
Rationale: we want to aim for the simplicity of Mustache templates, where the `if` condition in a template has only `if single_boolean_condition`. Moving the feature tests into a centralised module will make it easier to monitor the growth in complexity.